include/nvexec/multi_gpu_context.cuh

```diff
@@ -44,11 +44,11 @@ namespace nvexec {
         template <stdexec::__decays_to<R> Receiver>
         operation_state_t(Receiver&& rec)
           : rec_((Receiver&&) rec) {
-          status_ = STDEXEC_DBG_ERR(cudaStreamCreate(&stream_));
+          status_ = STDEXEC_CHECK_CUDA_ERROR(cudaStreamCreate(&stream_));
```

Comment:
Code around used to handle the error. It would seem that you are throwing an exception immediately instead. Most of these functions are used inside noexcept context. I'd prefer checking the error and completing with a `cudaError` instead of throwing an exception when possible.

Reply:
I dont think so, the main difference now is that we have the option to select the error handling
```cpp
  // If the `cudaError_t` parameter represents an error:
  // * If `STDEXEC_THROW_ON_CUDA_ERROR` is defined, an exception is thrown.
  // * If `STDEXEC_PRINT_ON_CUDA_ERROR` is defined, a diagnostic is printed to stdout.
  // * Otherwise, nothing happens.
```

By default we are not doing anything

Reply:
I agree with Georgii. Throwing here is never correct. Ditto for most of stdexec.